source/renderer/app/components/widgets/forms/PinCode.js

```diff
@@ -32,25 +32,31 @@ export default class PinCode extends Component<Props> {
   focusKey = 0;
   add = false;
 
-  onChange = (inputValue: ?number, key: number) => {
+  onChange = (inputValue: ?number | ?string, key: number) => {
     const { value, onChange } = this.props;
+
     const inputNewValue =
-      inputValue !== null && inputValue !== undefined
+      inputValue !== null && inputValue !== undefined && !Number.isNaN(inputValue)
```

Comment:
@aleksandardjordjeviciohk `Number.isNaN(inputValue)` will not work, either. Please, use `isNaN(inputValue)`